### PR TITLE
Use strategy instead of accelerator in pl

### PR DESCRIPTION
**Pull request by christofid**

### CODE REVIEW

**Comment by drugilsberg** [comment]
@christofid feel free to convert to ready for review once you think it's ready

File: requirements.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ minio==7.0.1
 modlamp>=4.0.0
 scikit-learn<0.24.0
 numpy>=1.16.5
-pytorch_lightning<=1.5.0
+pytorch_lightning==1.5.0
```
Comment (drugilsberg) [inline requirements.txt:11]: this is a bit too stringent, is there a way to work around this?

Comment (drugilsberg) [reply requirements.txt:11]: maybe with conditional blocks based on the lightning version?

**Review by drugilsberg — CHANGES_REQUESTED** [review]
See comment

**Comment by georgosgeorgos** [comment]
https://github.com/GT4SD/gt4sd-core/pull/149

**Comment by drugilsberg** [comment]
Closing in favour of #149

**Pull request closed without merging**